pipeline/pipeline.yml

```
@@ -129,109 +92,108 @@ resources:
   type: git
   icon: git
   source:
-    uri: ($ .cr_url $)
-    branch: ($ .cr_branch $)
-    ($- if eq .cr_cred_type "basic_auth" $)
-    username: ((($ .cr_cred_path $).username))
-    password: ((($ .cr_cred_path $).password))
+    uri: ($ .config_repository_url $)
+    branch: ($ .config_repository_branch $)
+    ($- if eq .config_repository_credential_type "basic_auth" $)
+    username: ((($ .config_repository_credential_path $).username))
+    password: ((($ .config_repository_credential_path $).password))
     ($- else $)
-    private_key: ((($ .cr_cred_path $).ssh_key))
+    private_key: ((($ .config_repository_credential_path $).ssh_key))
     ($- end $)
     paths:
-      - ($ .project $)/terraform/($ .environment $)/*
+      - ($ .config_root $)/terraform/*
 
 jobs:
-
 # Merge and trigger a plan whenever there is a commit in Terraform stack or config
 - name: terraform-plan
   serial: True
   max_in_flight: 1
   build_logs_to_retain: 10
   plan:
-    - do:
-      - get: git_stack-terraform
-        trigger: true
-      - get: git_config-terraform
-        trigger: true
-
-      - task: merge-stack-and-config
-        config:
-          <<: *merge-stack-and-config
-          inputs:
-          - name: git_config-terraform
-            path: "config"
-          - name: git_stack-terraform
-            path: "stack"
-        params:
-          CONFIG_PATH: ($ .project $)/terraform/($ .environment $)
-          STACK_PATH: terraform/($ $stack_path $)
-
-      - put: tfstate
-        params:
-          plan_only: true
-          terraform_source: merged-stack/
+  - do:
+    - get: git_stack-terraform
+      trigger: true
+    - get: git_config-terraform
+      trigger: true
+
+    - task: merge-stack-and-config
+      config:
+        <<: *merge-stack-and-config
+        inputs:
+        - name: git_config-terraform
+          path: "config"
+        - name: git_stack-terraform
+          path: "stack"
+      params:
+        CONFIG_PATH: ($ .config_root $)/terraform/
+        STACK_PATH: terraform/($ $stack_path $)
```

```suggestion
        STACK_PATH: terraform/($ $stack_path $) # ??
``` 
Le path semble incorrect ici, je ne vois pas de définition de var ici

---

here https://github.com/cycloid-community-catalog/stack-compute/blob/fe3887854066c45789194c3d5273c02eda58098e/pipeline/pipeline.yml#L18-L25